code/datums/achievements/achievements.dm

```diff
@@ -143,6 +144,12 @@
 	desc = "Trigger a keycard authentication device event, by yourself."
 	id = 23
 
+/datum/achievement/dab
```

Comment:
No.

Reply:
I do not control the rate at which spacemen dab

Reply:
Undocumented change, PR denied

Reply:
Please no, im downvoting this shit until this gets yeeted out